src/providers/ipa/ipa_deskprofile_rules_util.c

```diff
@@ -243,8 +244,10 @@ ipa_deskprofile_rules_create_user_dir(
         goto done;
     }
 
+    old_umask = umask(0022);
```

Comment:
Why do we need access by "other" ?

Reply:
If other can't access it, we can't create the desktop profile file ... at least according to my tests (tested using 750).

Reply:
This gives access to any user on the system to see rules files, that may not be ok.
What user/group own this directory at creation ?

Reply:
I guess all you really need here is just the x bit for others (traverse) if the owner is the root user